tensorflow/python/data/experimental/service/server_lib.py

```diff
@@ -39,7 +39,7 @@ class DispatcherConfig(
     port: Specifies the port to bind to. A value of 0 indicates that the server
       may bind to any available port.
     protocol: The protocol to use for communicating with the tf.data service.
-      Acceptable values include `"grpc" and "grpc+local"`.
+      Defaults to `"grpc"`.
```

Comment:
@aaudiber Were the changes to docstrings intentional?

Reply:
@jsimsa 

The "grpc+local" protocol was not being supported while testing using:

```python
def testStartDispatcherWithProtocolConfig(self):
    config = server_lib.DispatcherConfig(port=5000, protocol="grpc+local")
    dispatcher = server_lib.DispatchServer(config=config, start=False)
    dispatcher.start()
```
The test was erroring out with:

```console
RuntimeError: No credentials factory has been registered for protocol grpc+local. The available types are: [ grpc ]
```

Reply:
Yes, I originally planned to link the grpc+local protocol into the TF binary, but encountered issues using the protocol on windows and decided not to link it by default.